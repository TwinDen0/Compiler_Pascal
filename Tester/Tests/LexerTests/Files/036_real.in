1.003..1.23