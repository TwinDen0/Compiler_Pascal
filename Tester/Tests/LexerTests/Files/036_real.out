1	1	REAL	1,003	1.003
1	6	SEPARATOR	..	..
1	8	REAL	1,23	1.23
